roles/telemetry_verify_metrics/tasks/verify_mariadb_metrics.yml

```
@@ -0,0 +1,48 @@
+- name: Verify mariadb scrapeconfig exists
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_cr_test_id: RHOSO-xxxx
+    common_cr_list:
+      - kind: scrapeconfigs.monitoring.rhobs
+        name: telemetry-mariadb
+
+- name: Get mariadb CR names
+  ansible.builtin.shell: |
+    oc get mariadb -o custom-columns=NAME:.metadata.name --no-headers
+  register: cr_names
+  changed_when: false
+  failed_when: cr_names.rc >= 1 or cr_names.stdout == ""
+
+- name: Create mariadb pod list
+  ansible.builtin.set_fact:
+    common_pod_list: "{{ common_pod_list | default([]) + [item + '-server-0'] }}"
+  loop: "{{ cr_names.stdout_lines }}"
+
+- name: Verify mariadb pods are running
+  ansible.builtin.include_role:
+    name: common
+  vars:
+    common_pod_test_id: RHOSO-xxxx
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
+
+- name: Check mariadb metric endpoints
+  ansible.builtin.shell: |
+    oc rsh openstackclient curl https://{{ item }}.openstack.svc:9104/metrics
+  register: result
+  changed_when: false
+  failed_when: result.rc >= 1
+  loop: "{{ cr_names.stdout_lines }}"
+
+- name: |
+    TEST Use openstack observabilityclient to verify mariadb metrics are stored in prometheus
+    RHOSO-xxxx
+  ansible.builtin.shell: |
+    {{ openstack_cmd }} metric query mariadb_identity_info{instance=\'{{ item }}.openstack.svc:15691\'}
```

This is not a valid metric for the mysqld-exporter. You can use for example the following. And you'll need to use the `--disable-rbac` because the mysqld metrics don't have a project ID assigned.
```suggestion
    {{ openstack_cmd }} metric query --disable-rbac mysql_up{instance=\'{{ item }}.openstack.svc:3306\'}
```

---

fixed